tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1

```diff
@@ -13,11 +13,110 @@ param(
     [ValidateNotNullOrEmpty()]
     [String] $ClusterName,
     [Switch] $UseK8s=$false,
-    [string] $Tag
+    [string] $Tag,
+    # Temporary params for private bits
+    [Parameter(Mandatory=$true)]
+    [string] $connectedK8sPrivateWhlPath,
```

Comment:
We cannot let customer to enter this parameter if we ship this script to customers.

Reply:
this wont be merged to main (merged to a private integration branch already) - this is intended only for internal testing and needed to test private Arc bits.
the two parameters will not be needed in the final version and code around private bits will also be removed